venues/ICLR.cc/2018/Conference/python/fix-blinded-submissions.py

```diff
@@ -0,0 +1,74 @@
+'''
+This is a one time script to add missing ICLR 2018 blind submissions.
+'''
+
+import openreview
+import config
+import csv
+import argparse
+from collections import defaultdict
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print client.baseurl
+
+# get originals with missing blind copies
+submissions = client.get_notes(invitation=config.SUBMISSION)
+blind_submissions = client.get_notes(invitation=config.BLIND_SUBMISSION)
+
+print "there are {0} submissions and {1} blind copies".format(len(submissions), len(blind_submissions))
+
+blinds_by_original = {b.original: b for b in blind_submissions}
+
+missing_originals = [n.id for n in submissions if n.id not in blinds_by_original]
+
+# build the missing blind copies
+
+def get_bibtex(note):
+    first_word = note.content['title'].split(' ')[0].lower();
+
+    return '@article{\
+          \nanonymous2017' + first_word + ',\
+          \ntitle={' + note.content['title'] + '},\
+          \nauthor={Anonymous},\
+          \njournal={International Conference on Learning Representations},\
+          \nyear={2017}\
+      \n}'
+
+
+
+
+
+def get_blinded(original_note):
+    blind_submission_params = {
+        'original': original_note.id,
+        'invitation': config.BLIND_SUBMISSION,
+        'forum': None,
+        'signatures': [config.CONF],
+        'writers': [config.CONF],
+        'readers': ['everyone'],
+        'content': {
+            'authors': ['Anonymous'],
+            'authorids': ['ICLR.cc/2018/Conference/-/Paper{0}/Authors'.format(original_note.number)],
```

Comment:
I see that we are using the original note number here, but it is not what the submission process function does:

https://github.com/iesl/openreview-scripts/blob/master/venues/ICLR.cc/2018/Conference/process/submissionProcess.js#L108

Reply:
numbers of different invitations(original and blind) are not synced!!!

Reply:
I see. I can change this if you'd like, but I've been running scripts to make sure that the blind note and original notes have the same numbers

Reply:
ok, you already run this script and it worked so I'll merge it.

Reply:
the reason the blind/original are not synced is because there are two withdrawn notes after the deadline. withdrawing after the deadline deletes the blind, keeps the original, and creates a "withdrawn submission"